tools/cleanup/cleanup.go

```diff
@@ -44,7 +44,7 @@ import (
 
 var (
 	// Authentication method for using Google Cloud Registry APIs.
-	auther = authn.DefaultKeychain
+	author = authn.DefaultKeychain
```

Comment:
This is not author. :)

```suggestion
	auther = authn.DefaultKeychain
```